src/idsk/components/search-results/template.njk

```diff
@@ -0,0 +1,189 @@
+<div class="idsk-search-results {{ params.classes if params.classes }}" data-module="idsk-search-results">
```

Comment:
celkovo si nemyslim zeby toto malo byt medzi komponentami, malo by to byt vo vzoroch, komponenty by mali byt len filtre 3 typy (radio, datum, checkbox).. kedze ale termin bol nastaveny nie podla casu kolko trva vyvoj ale bol zrazu dany, musim akceptovat toto. V ramci update-u by som to ale neskor premiestnil tam kam to patri